packages/eslint-plugin/docs/rules/no-throw-literal.mdx

```diff
@@ -16,10 +16,10 @@ This rule restricts what can be thrown as an exception.
 
 :::warning
 This rule is being renamed to [`only-throw-error`](./only-throw-error.mdx).
-When it was first created, it only prevented literals from being thrown (hence the name), but it has now been expanded to only allow expressions which have a possibility of being an `Error` object.
-With the `allowThrowingAny` and `allowThrowingUnknown` options, it can be configured to only allow throwing values which are guaranteed to be an instance of `Error`.
+The current name, `no-throw-literal`, will be removed in a future major version of typescript-eslint.
```

Comment:
since this PR changes multiple rules, maybe we should also change its title?
(so it'll also be squashed with the proper commit message)

or maybe split tp multiple PRs?

Reply:
This comes out of https://github.com/typescript-eslint/typescript-eslint/pull/8821#discussion_r1566177427. 
I think it makes sense to include in this change set since it's such a small (and related) change.